<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonOpenDialog">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Dialog</string>
    </property>
   </widget>
   <widget class="plotter" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>99</y>
      <width>591</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>156</width>
      <height>44</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="2" column="0">
      <widget class="QSlider" name="horizontalSliderRaio">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLCDNumber" name="lcdNumber_7"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>   RAIO ESFERA</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>591</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButtonChooseColor">
       <property name="text">
        <string>Cor</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>ELLIPSOID</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButtonColor">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="6">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>R</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSlider" name="horizontalSliderR">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLCDNumber" name="lcdNumber_4"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>G</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSlider" name="horizontalSliderG">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLCDNumber" name="lcdNumber_5"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>B</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSlider" name="horizontalSliderB">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLCDNumber" name="lcdNumber_6"/>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>X</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSlider" name="horizontalSliderBoxX">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLCDNumber" name="lcdNumber_11"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Y</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSlider" name="horizontalSliderBoxY">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLCDNumber" name="lcdNumber_12"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Z</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSlider" name="horizontalSliderBoxZ">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLCDNumber" name="lcdNumber_13"/>
         </item>
        </layout>
       </item>
       <item row="0" column="2">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>RX</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSlider" name="horizontalSliderERx">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="2" colspan="2">
          <widget class="QLCDNumber" name="lcdNumber_8"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>RY</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QSlider" name="horizontalSliderERy">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLCDNumber" name="lcdNumber_9"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>RZ</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QSlider" name="horizontalSliderERz">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLCDNumber" name="lcdNumber_10"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="2" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>BOX</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>188</width>
      <height>83</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Dim X</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="2">
      <widget class="QSlider" name="horizontalSliderDimX">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2" rowspan="2">
      <widget class="QLCDNumber" name="lcdNumber"/>
     </item>
     <item row="1" column="0" rowspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Dim Y</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="horizontalSliderDimY">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLCDNumber" name="lcdNumber_2"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Dim Z</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="horizontalSliderDimZ">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLCDNumber" name="lcdNumber_3"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>Add</string>
     </property>
     <addaction name="actionVoxel"/>
     <addaction name="actionBox"/>
     <addaction name="actionSphere"/>
     <addaction name="actionEllipsoid"/>
    </widget>
    <widget class="QMenu" name="menuDelete">
     <property name="title">
      <string>Delete</string>
     </property>
     <addaction name="actionVoxel_cut"/>
     <addaction name="actionBox_cut"/>
     <addaction name="actionSphere_cut"/>
     <addaction name="actionEllipsoid_cut"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="menuNew"/>
    <addaction name="menuDelete"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionSave"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/rec/img/icons8-save-48.png</normaloff>:/rec/img/icons8-save-48.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/rec/img/icons8-delete-48.png</normaloff>:/rec/img/icons8-delete-48.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionVoxel">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/rec/img/icons8-add-48.png</normaloff>:/rec/img/icons8-add-48.png</iconset>
   </property>
   <property name="text">
    <string>Voxel</string>
   </property>
  </action>
  <action name="actionBox">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/rec/img/icons8-add-48.png</normaloff>:/rec/img/icons8-add-48.png</iconset>
   </property>
   <property name="text">
    <string>Box</string>
   </property>
  </action>
  <action name="actionSphere">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/rec/img/icons8-add-48.png</normaloff>:/rec/img/icons8-add-48.png</iconset>
   </property>
   <property name="text">
    <string>Sphere</string>
   </property>
  </action>
  <action name="actionEllipsoid">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/rec/img/icons8-add-48.png</normaloff>:/rec/img/icons8-add-48.png</iconset>
   </property>
   <property name="text">
    <string>Ellipsoid</string>
   </property>
  </action>
  <action name="actionVoxel_cut">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/rec/img/icons8-trash-can-48.png</normaloff>:/rec/img/icons8-trash-can-48.png</iconset>
   </property>
   <property name="text">
    <string>Voxel</string>
   </property>
  </action>
  <action name="actionBox_cut">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/rec/img/icons8-trash-can-48.png</normaloff>:/rec/img/icons8-trash-can-48.png</iconset>
   </property>
   <property name="text">
    <string>Box</string>
   </property>
  </action>
  <action name="actionSphere_cut">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/rec/img/icons8-trash-can-48.png</normaloff>:/rec/img/icons8-trash-can-48.png</iconset>
   </property>
   <property name="text">
    <string>Sphere</string>
   </property>
  </action>
  <action name="actionEllipsoid_cut">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/rec/img/icons8-trash-can-48.png</normaloff>:/rec/img/icons8-trash-can-48.png</iconset>
   </property>
   <property name="text">
    <string>Ellipsoid</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/rec/img/icons8-libre-office-suite-48.png</normaloff>:/rec/img/icons8-libre-office-suite-48.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>plotter</class>
   <extends>QWidget</extends>
   <header>plotter.h</header>
   <container>1</container>
   <slots>
    <slot>setR(int)</slot>
    <slot>setG(int)</slot>
    <slot>setB(int)</slot>
    <slot>setX(int)</slot>
    <slot>setY(int)</slot>
    <slot>setZ(int)</slot>
    <slot>setRadius(int)</slot>
    <slot>setRx(int)</slot>
    <slot>setRy(int)</slot>
    <slot>setRz(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>horizontalSliderDimX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderDimY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderDimZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_3</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderR</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_4</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderG</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_5</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>461</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_6</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>492</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>495</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRaio</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_7</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderERx</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_8</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>594</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>696</x>
     <y>417</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderERy</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_9</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>600</x>
     <y>452</y>
    </hint>
    <hint type="destinationlabel">
     <x>698</x>
     <y>460</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderERz</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_10</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>600</x>
     <y>489</y>
    </hint>
    <hint type="destinationlabel">
     <x>700</x>
     <y>489</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderR</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget</receiver>
   <slot>setColor_r(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderG</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget</receiver>
   <slot>setColor_g(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget</receiver>
   <slot>setColor_b(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>493</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderDimX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget</receiver>
   <slot>setX(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderDimY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget</receiver>
   <slot>setY(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderDimZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget</receiver>
   <slot>setZ(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRaio</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget</receiver>
   <slot>setRadius(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderERx</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget</receiver>
   <slot>setRx(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>524</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderERy</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget</receiver>
   <slot>setY(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderERz</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget</receiver>
   <slot>setRz(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>528</x>
     <y>492</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderBoxX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_11</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>426</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderBoxY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_12</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderBoxZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_13</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>492</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>494</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderR</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>setRed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>31</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderG</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>setGreen(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>23</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>setBlue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setRed(int)</slot>
  <slot>setGreen(int)</slot>
  <slot>setBlue(int)</slot>
 </slots>
</ui>
